src/edit.c

```diff
@@ -2963,12 +2986,23 @@ stuff_inserted(
     return OK;
 }
 
-    char_u *
+    string_T *
 get_last_insert(void)
 {
-    if (last_insert == NULL)
-	return NULL;
-    return last_insert + last_insert_skip;
+    static string_T insert = {NULL, 0};
```

Comment:
I don't think using a static variable is a good idea. If a caller modifies the return value (like what is done in `stuff_inserted()`), it'll have to take extra precaution not to call `get_last_insert()`, as that'll change the value back.